src/WSManProcessor.ts

```diff
@@ -269,18 +269,23 @@ export class WSManProcessor {
     }
   }
 
-  async put (clientId: string, action: string, obj: any, amtuser?: string, amtpass?: string): Promise<void> {
+  async put (clientId: string, action: string, obj: any, amtuser?: string, amtpass?: string, noselectors: boolean = false): Promise<void> {
     const clientObj: ClientObject = this.clientManager.getClientObject(clientId)
     try {
       const amtstack = this.getAmtStack(clientId, amtuser, amtpass)
-      await amtstack.Put(action, obj, (stack, name, jsonResponse, status) => {
+      const callback = (stack, name, jsonResponse, status): any => {
```

Comment:
i'm not seeing a return --- wouldn't it be void?

Reply:
done